imports/api/chain/server/methods.js

```diff
@@ -89,6 +89,7 @@ Meteor.methods({
                     console.log(e);
                 }
 
+		if (Meteor.settings.public.mintingDenomination) {
```

Comment:
Should it be `Meteor.settings.public.mintingDenom` here or you are adding a new setting?

Reply:
You're right, my mistake!